.github/workflows/test.yml

```diff
@@ -55,6 +56,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+          submodules: recursive
```

Comment:
This PR should fix the minion issues: https://github.com/conjure-cp/conjure-oxide/pull/1151